src/Compilers/Core/CodeAnalysisTest/AnalyzerFileReferenceTests.cs

```diff
@@ -16,11 +16,13 @@ namespace Microsoft.CodeAnalysis.UnitTests
 {
     public class AnalyzerFileReferenceTests : TestBase
     {
-        private static SimpleAnalyzerAssemblyLoader _analyzerLoader = new SimpleAnalyzerAssemblyLoader();
 
         public static AnalyzerFileReference CreateAnalyzerFileReference(string fullPath)
         {
-            return new AnalyzerFileReference(fullPath, _analyzerLoader.LoadFromPath);
+            var bytes = File.ReadAllBytes(fullPath);
+            var assembly = Assembly.Load(bytes);
```

Comment:
Eventually our tests are going to need to run on CoreCLR.  Assembly loading is going to be one of the aspects affected by this.  If there are other places where we load assemblies (suspecting their are) could we centralize this to a single location to make it easy to change later?